src/config/backend/EdFi.DmsConfigurationService.Backend/Claims/Claims.json

```diff
@@ -0,0 +1,8662 @@
+{
```

Comment:
These are the "Embedded" claims, literally meaning this file is an embedded resource. This was taken from the `0010-Insert-Claimset.sql` file for the `claimSets` section and the `AuthorizationHierarchy.json` file in `CmsHierarchy/ClaimSetFiles` for the `claimsHierarchy` section. When CMS is in ClaimsSource=Embedded mode these claims are loaded when the DB is empty and whenever a reload command is issued.